### PR TITLE
Update ion range slider example

### LINKED ISSUE #12494: Ion slider extension example broken
cc @mattpap it looks like this

Screen Shot 2022-10-17 at 19 00 40

### PR DESCRIPTION
**Pull request by mattpap**
fixes #12494

### CODE REVIEW

**Comment by bryevdv** [comment]
@mattpap I have resolved the small conflict from the changes in my examples cleanup PR

File: examples/advanced/extensions/widget.py [hunk 1]
```diff
@@ -79,7 +73,6 @@ class IonRangeSlider(InputWidget):
     y = Array.from(x, (x) => Math.pow(x, pow))
```
Comment (mattpap) [inline examples/advanced/extensions/widget.py:73]: What's up with this code? What is `x` and `y`?

Comment (bryevdv) [reply examples/advanced/extensions/widget.py:73]: oops, missing `const` in front of both

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
I will go ahead and merge this in order to get it in an rc4 tonight.

**Pull request merged**